base/arrayshow.jl

```diff
@@ -427,6 +427,7 @@ end
 
 ### 0-dimensional arrays (#31481)
 show_zero_dim(io::IO, X::BitArray{0}) = print(io, "BitArray(", Int(X[]), ")")
+print("hi")
```

Comment:
:)

Reply:
I am sorry, I was checking the flow , correcting it right now